src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/FileUploadsValidatorServiceTests.cs

```diff
@@ -255,22 +259,23 @@ await using (var context = InMemoryApplicationDbContext(contextId))
         [Fact]
         public async Task ValidateDataFilesForUpload_ReplacingDataFileWithFileOfSameName()
         {
-            var releaseId = Guid.NewGuid();
+            var release = _fixture.DefaultRelease()
+                .Generate();
+
+            // The file being replaced here has the same name as the one being uploaded, but that's ok.
+            var fileBeingReplaced = _fixture.DefaultFile()
+                .WithFilename("test.csv");
```

Comment:
> Looks like you got away without the need for .Generate() here. Wonder where else we could make use of this?

I've made some changes in 2459a4ae32e4e56e5211f5326af6747e71ed6671 to remove other calls to `.Generate()` but only in `FileUploadsValidatorServiceTests`.

I've done this on another branch too for the codebase but it's turned into a much bigger change so I will raise that as a separate PR.

Reply:
>Should this file being replaced not be stored in the db prior to carrying out the method under test? I guess it wasn't in the original and it doesn't matter so much because it's just being passed in as an argument rather than being looked up. No worries!

I think the change I already made here will save it in the database because it should be saved as part of saving the `ReleaseFile`. 

I've made another change a bit further down to `ValidateDataArchiveEntriesForUpload_ReplacingDataFileWithFileOfSameName` so that we do the same thing there. That's commit ce22547707462e294ba7fa512bf74eb0a678c80a. Thanks.

Reply:
Ahh gotcha, thanks!